### ISSUE TITLE
Update facility API to allow filtering by extended fields 

### ISSUE THREAD

**Issue by jwalgran** [open-apparel-registry/open-apparel-registry#1562]
## Overview

Update facility API to allow filtering by extended fields 

[Draft API documentation](https://docs.google.com/document/d/1ukEfgQtV6n_EFbGd8z7KvBzNMxg0U-FbxGz-lyysogo/edit)

### Describe the solution you'd like

- [x]   Depends on the extended field values being added to the facility index
- [ ]   Reference the search index updates( #1651). Update FacilityManager.filter_by_query_params to handle the extended fields
    -   Multiple values for a field are “OR”ed together
    -   For parent company search parent_company_name or, if a submitted value is numeric, search parent_company_id
    -   For product_type `clean` values before searching
    -   For native_language_name use `unidecode` on the query value and do a substring match on the column
- [ ]   Update the swagger docs

**Comment by TaiWilkin** [comment]
In the draft API documentation we said we would use the format ‘number_of_workers=500’ and ’number_of_workers=‘0,500’ for this field. However, when we set up the facility index we set it up to translate the values into the standard ranges and include those in the index instead of the exact values. 
My first thought was to translate the API-submitted ranges into the standard ranges, then match that way (ie: if someone submits '500,2000' we match values with ranges 'Less than 1000' or '1001-5000'). This causes weird behavior though (the search above would return a facility with 250 workers, since there are less than 1000 workers, even though the minimum range value sent is 500). 
I think we're going to need to change the expected input to require users to submit the standard ranges directly (ie `number_of_workers=Less%20than%201000`) even though this might be a little less intuitive for the Swagger endpoint / API requests. 
@jwalgran Does this seem like a reasonable plan?
